test/ontarget/S32K146EVB/src/LPUART.c

```diff
@@ -42,10 +42,9 @@ void LPUART1_init(void)  /* Init. summary: 9600 baud, 1 stop bit, 8 bit format,
 	PCC->PCCn[PCC_LPUART1_INDEX] |= PCC_PCCn_PCS(0x02)    /* Clock Src = 2 (SIRCDIV2_CLK) */
                             	 |  PCC_PCCn_CGC_MASK;     /* Enable clock for LPUART1 regs */
 
-	// TODO: 115200
-	LPUART1->BAUD = LPUART_BAUD_SBR(0x34)  	/* Initialize for 9600 baud, 1 stop: */
-                	|LPUART_BAUD_OSR(15);  	/* SBR=52 (0x34): baud divisor = 8M/9600/16 = ~52 */
-											/* OSR=15: Over sampling ratio = 15+1=16 */
```

Comment:
![image](https://user-images.githubusercontent.com/3298404/57198026-d6ad0c80-6f76-11e9-973d-95a03dc351e9.png)

Reply:
Oh geez NXP. I'll fix this. Sorry (I didn't do it!)